src/main/views/viewManager.ts

```diff
@@ -89,6 +89,13 @@ export class ViewManager {
         view.once(LOAD_FAILED, this.failLoading);
     }
 
+    reloadViewIfNeeded = (viewName: string) => {
+        const view = this.views.get(viewName);
+        if (!view?.getWebContents()?.getURL().startsWith(view.tab.url.toString())) {
+            view?.load(view.tab.url);
```

Comment:
my only worry here is that we trigger a loop somehow,
go to base url, that triggers the `__login__` storage event, which reloads the base url, etc.
but we can't listen to the event only once, because someone might logout and login again.

so I'm not proposing to change it, but just raise a bit of awareness in case it is a real possibility

Reply:
I'm pretty sure the `__login__` event is only thrown once so this shouldn't be a concern